### ISSUE TITLE
Training Time with 8 NVIDIA RTX 3090 GPUs

### ISSUE THREAD

**Issue by VisionLover** [haoningwu3639/StoryGen#29]
Hi Haoning,
I noticed that you utilized 8 NVIDIA RTX 3090 GPUs for training your model. Could you please share how much time was spent training the model with these resources?
Thank you for your time and assistance.

**Comment by haoningwu3639** [comment]
Since our model is built upon pre-trained Stable Diffusion, the training is actually quite fast.
Sometimes we only use 4 * 3090 with a larger number of gradient accumulation.
The first stage may only take half a day to adapt the self-attention layers.
The second stage needs to extract context features to train our context module, which will be a little bit slower.
But it should take about one day to train with a single previous frame as context, and maybe another half day to further fine-tune with more contexts.
